### PR TITLE
Add release interview for 1.22

### PR DESCRIPTION
**Pull request by craigbox**
Publish an abridged transcript of the Kubernetes Podcast interview with Savitha Raghunathan as a kubernetes.io blog post.

See also: #29146, #22882, #22716, #17963 et al.

/cc @savitharaghunathan

### CODE REVIEW

File: content/en/blog/_posts/2021-12-01-kubernetes-1.22-release-interview.md [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+
+SAVITHA RAGHUNATHAN: No, unfortunately. That is a question that everyone asks. I have not used Matlab. I haven't used it even now. I don't use it for work. I didn't have any necessity for my school work. I didn't have anything to do with Matlab. I never analyzed, or did data processing, or anything, with Matlab. So unfortunately, no.
+
+Everyone asks me like, you're working at [MathWorks](https://en.wikipedia.org/wiki/MathWorks). Have you used Matlab? I'm like, no.
+
+**CRAIG BOX: Fair enough. Nor have I. But it's been around since the late 1970s, so I imagine there are a lot of people who will have come across it at some point. Do you work with a lot of people who have been working on it the whole time?**
+
+SAVITHA RAGHUNATHAN: Kind of. Not all the time, but I get to meet some folks who work on the product itself. Most of my interactions are with the infrastructure team and platform engineering teams at MathWorks. One other interesting fact is that when I joined the company-- MathWorks has extensive internal curriculum for training and learning, which I really love. They have intuitive Matlab, and that's on my bucket of things to do.
```
Comment (savitharaghunathan) [inline content/en/blog/_posts/2021-12-01-kubernetes-1.22-release-interview.md:83]: nit: intuitive Matlab -> MatLab training course

File: content/en/blog/_posts/2021-12-01-kubernetes-1.22-release-interview.md [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+
+I started out because I wanted to make these little toys. They call them [amigurumi](https://en.wikipedia.org/wiki/Amigurumi) in the crochet world. I wanted to make them. That's why I started out. I'm trying. I think I made [a little cat](https://twitter.com/KubernetesPod/status/1423188323347177474/photo/2) which doesn't look like a cat, but it is a cat. I have to tell everyone that it's a cat so that they don't mock me later, but.
+
+**CRAIG BOX: It's an artistic interpretation of a cat.**
+
+SAVITHA RAGHUNATHAN: It definitely is.
+
+---
+
+_[Savitha Raghunathan](https://twitter.com/coffeeartgirl) is a Senior Platform Engineer at MathWorks and served as the Kubernetes 1.22 release team lead._
```
Comment (savitharaghunathan) [inline content/en/blog/_posts/2021-12-01-kubernetes-1.22-release-interview.md:339]: Senior Platform Engineer at MathWorks -> Senior Software Engineer at Red hat

Comment (craigbox) [reply content/en/blog/_posts/2021-12-01-kubernetes-1.22-release-interview.md]: Congratulations!

File: content/en/blog/_posts/2021-12-01-kubernetes-1.22-release-interview.md [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+
+Sometimes I have learned more leadership skills from my release lead shadows. They are awesome, and they are mature. I constantly learn from them, and I admire them a lot.
+
+It also helps to have good, strong individuals in the team who can step up and help when needed. For example, unfortunately, we lost one of our teammates after the start of the release cycle. That was tragic. His name was [Peeyush Gupta](https://github.com/cncf/memorials/blob/main/peeyush-gupta.md). He was an awesome and wonderful human-- very warm.
+
+I didn't get more of a chance to interact with him. I had exchanged a few Slack messages, but I got his warm personality. I just want to take a couple of seconds to remember him. He was awesome.
+
+After we lost him, we had this strong person from the team step up and lead the communications, who had never been a part of the release team before at all. He was a shadow for the first time. His name was Jesse Butler. So he stepped up, and he just took it away. He ran the comms show for 1.22.
```
Comment (savitharaghunathan) [inline content/en/blog/_posts/2021-12-01-kubernetes-1.22-release-interview.md:297]: nit: name was -> name is

**Review by savitharaghunathan — COMMENTED** [review]
Thanks, Craig. Just one nit, otherwise lgtm from me.

**Comment by reylejano** [comment]
The blog project board shows nothing scheduled for 12/1 (today)
Other than the nit @savitharaghunathan pointed out
/lgtm

File: content/en/blog/_posts/2021-12-01-kubernetes-1.22-release-interview.md (hunk 3 above)
Comment (craigbox) [reply content/en/blog/_posts/2021-12-01-kubernetes-1.22-release-interview.md]: Nit fixed.

**Comment by craigbox** [comment]
Nit fixed, ready for /lgtm again.  Thanks!

**Comment by reylejano** [comment]
re-applying lgtm
/lgtm

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm 
(content side)

**Review by sftim — COMMENTED** [review]
Oops, this review didn't send before.

**Comment by sftim** [comment]
/approve

@craigbox if you agree with any of my nits, you're very welcome to open a PR that fixes them up.

**Comment by craigbox** [comment]
I might just, @sftim, but only because I literally just watched this:
https://mobile.twitter.com/yayalexisgay/status/1465776683819094018?s=12

**Pull request merged**